src/mylinef.cpp

```diff
@@ -290,19 +292,22 @@ MyLineF::intersects_crossHypot(const QLineF& l, QPointF* intersectionPoint) cons
 QLineF::IntersectionType
 MyLineF::intersects_flsiOrigX(const QLineF &l, QPointF *intersectionPoint) const
 {
+	constexpr qreal tolerance = 1000;
```

Comment:
This is our tolerance to parallel lines in epsilons of relative precision, which is definitely what we should use instead of the absolute deviation. A long vector scales the absolute deviation appropriately @:308 where the epsilon is calculated.